.github/workflows/test.yml

```diff
@@ -175,6 +175,7 @@ jobs:
 
       # Workaround for issue https://github.com/actions/upload-artifact/issues/174
       - name: Upload test results
+        if: always()
```

Comment:
@andonyns with a test failure above causing exit code 1 from jest, this step was not running before.